test-requirements.txt

```diff
@@ -2,5 +2,5 @@ bashate
 flake8==6.0.0
 flake8-import-order==0.18.2
 stestr
-pylint==2.17.4
+pylint==3.0.0
```

Comment:
The default on Noble I see is `3.0.3`. Is there a reason not to use that?

Reply:
No, I figured I'd start very gently :smile:

Reply:
Ah, I didn't realise this is a draft!

Reply:
I am getting lots of errors at this point and I am trying to figure out how to fix those.

Reply:
if you want to do it it'd be fine to do as a separate patch that makes it clear in the commit that no real changes are being made.